public/style.css

```diff
@@ -121,33 +121,177 @@ main {
 }
 
 header {
-    width: 100%;
-    height: 50px;
     background-color: var(--primary-color-3);
     border-bottom: var(--border-style-black);
-    
+    width: 100%;
+}
+
+header div {
+    max-width: 768px;
+    margin: 0 auto; 
     display: flex;
-    flex-direction: row;
     align-items: center;
+    justify-content: space-between;
+    padding: 1em;
+}
+
+header div a img {
+    width: 10em;
+}
+
+.navbar-menu {
+    position: fixed;
+    top: 0;
+    right: -100vw;
+    width: 100%;
+    height: 100vh;
+    background: #ededed;
+    transition: right 0.5s;
+    overflow-y: auto;
+    max-width: 768px;
+
+    display: flex;
+    flex-direction: column;
     justify-content: center;
-    padding: 1em 0;
+    align-items: center;
 }
 
-header div {
-    max-width: 1200px;
+.navbar-menu li{
+    font-size: 3em;
+    line-height: auto;
+    text-transform: none;
 }
 
-header img {
-    width: 10em;
-    height: auto;
+.navbar-menu a{
+    color: var(--primary-color-1);
 }
 
-header ul,
-header input {
-    display: none;
+.navbar-donate {
+    color: var(--active-color-1);
+    font-weight: 900;
 }
 
+#navbar-toggle .navbar-menu-button {
+  display: block;
+  width: 33px;
+  height: 4px;
+  margin-bottom: 5px;
+  position: relative;
+  
+  background: var(--primary-color-1);
+  border-radius: 3px;
+  
+  z-index: 1;
+  
+  transform-origin: 4px 0px;
+  
+  transition: 
+  transform 0.5s cubic-bezier(0.77,0.2,0.05,1.0), 
+  background 0.5s cubic-bezier(0.77,0.2,0.05,1.0),
```

Comment:
Hier kan je je code meer DRY maken met een custom property.

Reply:
Is vergeet altijd dat dit kan, het is eigenlijk best handig. :) 

https://github.com/SamaraFellaDina/pleasurable-ui/commit/7f4c173a27f1a4ae2f317d464ed6602c19f39734